app/static/src/styles/comments.css

```diff
@@ -0,0 +1,15 @@
+.s26-comment-group {
+    background-color: #ffffff;
+    padding: 25px 25px 5px;
+    margin-top: 15px;
+    border: 2px #1d70b8 solid;
+}
+
+.s26-comment-group:last-of-type {
+    margin-bottom: 30px;
+}
+
+.s26-comment-group .govuk-body-s {
```

Comment:
s26 is a prefix to show the sprint from the prototype - remove all s26 prefixes here